torch/_dynamo/variables/builder.py

```diff
@@ -397,7 +397,7 @@ def index_source(key):
             return SkipFilesVariable(
                 value, guards=make_guards(GuardBuilder.FUNCTION_MATCH)
             )
-        elif istype(value, (type, ABCMeta)):
+        elif istype(value, (type, ABCMeta)) and not issubclass(value, torch.nn.Module):
```

Comment:
does isinstance work here?

Reply:
value in this case is a class name, not instance.